cli/render.go

```diff
@@ -282,6 +282,10 @@ func printCurrentFieldsInfo(fields []*report.Field) {
 
 		fmt.Print(strings.Repeat("  ", int(counter+1)))
 
+		for ; counter%field.Size != 0; counter++ {
```

Comment:
We are not using inline variable changing inside the loops, could you please update your code to something like this:

```go
		for counter%field.Size != 0 {
			fmtc.Printf("{r}□ {!}")
      counter++
		}
```

Reply:
yes immediately modify. thank you very much.